test/onnx/gen_onnx.py

```diff
@@ -1545,6 +1591,72 @@ def convinteger_bias_test():
     return ([node], [x, y, z], [out])
 
 
+@onnx_test()
```

Comment:
Could you please change one of the inputs, x or y to be UINT8. Mixed types are allowed by the standard, I think. You can either modify the test, or add a fresh one. Thanks.

Reply:
I have the case below with convinteger_mismatched_input_types_test() but I can add a case for dual bias of different types

Reply:
Added the extra case of uint8 for one input and bias and int8 for another by reusing one of the test and changing the types. Gold data should be the same since the quantizations between int8/uint8 shouldn't effect the intended information/accumulation at the end when we get an int32 value